src/app/core/session/auth/keycloak/keycloak-auth.service.ts

```diff
@@ -1,20 +1,15 @@
-import { AuthService } from "../auth.service";
 import { Injectable } from "@angular/core";
-import Keycloak from "keycloak-js";
-import {
-  HttpClient,
-  HttpErrorResponse,
-  HttpHeaders,
-  HttpStatusCode,
-} from "@angular/common/http";
-import { firstValueFrom, Observable } from "rxjs";
+import { HttpClient } from "@angular/common/http";
+import { firstValueFrom, NEVER, Observable } from "rxjs";
 import { parseJwt } from "../../../../utils/utils";
 import { environment } from "../../../../../environments/environment";
-import { AuthUser } from "../../session-service/auth-user";
-import { catchError } from "rxjs/operators";
+import { AuthUser } from "../auth-user";
+import { KeycloakService } from "keycloak-angular";
+import { SessionType } from "../../session-type";
 
 @Injectable()
-export class KeycloakAuthService extends AuthService {
```

Comment:
my heart is still bleeding a tiny bit for losing the generic AuthService interface. But I think there is not too much value in keeping that (with the exact same interface as this KeycloakAuthService) just for the sake of abstraction ...